app/controllers/sessions_controller.rb

```diff
@@ -184,6 +184,6 @@ def record_mfa_login_duration(mfa_type:)
     started_at = Time.zone.parse(session[:mfa_login_started_at]).utc
     duration = Time.now.utc - started_at
 
-    StatsD.distribution("login.mfa.#{mfa_type}.duration", duration, sample_rate: 1.0)
+    StatsD.distribution("login.mfa.#{mfa_type}.duration", duration)
```

Comment:
Isn't the key in here to just use measure block to collect durations?

https://github.com/Shopify/statsd-instrument#statsdmeasure

Reply:
Good point Josef! I think we can use `measure` instead since we are [emitting a timing metric](https://github.com/Shopify/statsd-instrument/blob/master/lib/statsd/instrument/client.rb#L207). I believe we'd still need to manually calculate the duration as opposed to delegating the calculation to a block because we're measuring the duration between the end of the `SessionsController#create` action until just before the `do_login` method in either `SessionsController#mfa_create` or `SessionsController#webauthn_create`.